geode-core/src/main/java/org/apache/geode/internal/cache/PrimaryBucketLockException.java

```diff
@@ -19,7 +19,7 @@
 import org.apache.geode.GemFireException;
 
 /**
- * Thrown by {@link PartitionedRegion#computeWithPrimaryLocked(Object, Runnable)}
+ * Thrown by {@link PartitionedRegion#computeWithPrimaryLocked(Object, Runnable, boolean)}
```

Comment:
this javadoc needs to be updated for the signature with no boolean

Reply:
Truth. Thanks, Darrel!